### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by nevisende**
- Initialised React app.
- Added React Redux (`npm install react-redux).
- Structured my application files using a "feature folder" approach and use the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux) for my Redux files. My files/folder structure looks like this:
    ```    
    ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | configureStore.js
        | index.js
    ``` 
- The *building blocks* of my app were set as re-usable components.
- Added [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation:
    - Books - the default view
        - Displayed the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)
        - Included a form for adding a book (no functionality yet)
    - Categories
        - Displayed "Under construction" text only.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <BrowserRouter>
+    <ul>
+      <img src={logo} alt="logo" />
+      <li>
+        <Link to="/">Books</Link>
+      </li>
+      <li>
+        <Link to="categories">Categories</Link>
+      </li>
+    </ul>
```
Comment (Emmyn5600) [inline src/index.js:25]: - [ ] I recommend using the app.js file to keep both links and routes as well as wrap the app.js inside the index.js file.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <ul>
+      <Book />
+      <Book />
+      <Book />
+      <Book />
+      <Book />
+    </ul>
+  );
+}
```
Comment (Emmyn5600) [inline src/redux/books/books.js:13]: - Please implement the following as it is listed in the project specifications.

setup-13

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @nevisende  

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️

- Check the comment under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/redux/books/books.js (hunk 2 above)
Comment (nevisende) [reply src/redux/books/books.js:13]: Man, please read right specification please.

Comment (nevisende) [reply src/redux/books/books.js:13]: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md

**Comment by mke2111** [comment]
Project Under Second Opinion review.
Kindly wait patiently for a response. 🙏🏽

**Review by mke2111 — COMMENTED** [review]
Hello @nevisende  👋,

_This is an additional review as a result of the second opinion request._

Well done with your progress so far.

> I'll try and explain everything just so that we are sure we all agree with the project requirements.

The default view of books should list all the books added. To achieve this, you need to add a component necessary for displaying all the books (you may call this BookList), well done on passing the Book component to this component. You have this component currently in your redux directory. It would be best if this file is inside the components directory. For you to pass down the state from the Book component to the BookList component, you need to use at least `props` so that the Book component renders the details of the book.
Furthermore, the project requirements state that you need to have a form for adding a new book. Add this feature inside the 'BookList' component. 
Remember, all functionality is not required at this point. So the book form should just be there even if it doesn't submit the books successfully. The same applies to the BookList component.

Make the requested changes earlier and submit another CRR once you are done. Don't hesitate to reach out to any of the reviewers for more clarification/help, remember to tag the CRs in this thread. 

Keep rocking 🚀 & Happy Coding 💻

**Review by omar-labana — APPROVED** [review]
Hi @nevisende, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**